Xplat/src/main/java/vazkii/botania/common/block/block_entity/ManaEnchanterBlockEntity.java

```diff
@@ -465,6 +473,14 @@ public static Direction.Axis canEnchanterExist(Level world, BlockPos pos) {
 		};
 	}
 
+	private static Rotation getAxisRotation(Direction.Axis axis) {
+		return switch (axis) {
+			case X -> Rotation.NONE;
+			case Z -> Rotation.CLOCKWISE_90;
+			default -> null;
```

Comment:
you could just throw here and elide the null check on the caller side, this is never going to be Y